src/trio/_util.py

```diff
@@ -247,11 +247,9 @@ def decorator(func: CallT) -> CallT:
         func.__name__ = attr_name
         func.__qualname__ = ".".join((cls.__qualname__, attr_name))
 
-        func.__doc__ = """Like :meth:`~{}.{}.{}`, but async.
+        func.__doc__ = f"""Like :meth:`~{wrapped_cls.__module__}.{wrapped_cls.__qualname__}.{attr_name}`, but async.
 
-        """.format(
-            wrapped_cls.__module__, wrapped_cls.__qualname__, attr_name
-        )
+        """
```

Comment:
Were these extra lines ever intentional?

Reply:
Removed in ec4c012